### PR TITLE
DISCO-1397 handle 404 from org users endpoint

### PR DESCRIPTION
**Pull request by xnick421**
Allow 404 to bypass Promise.all catch so that
we gracefully display available users

### CODE REVIEW

File: src/components/PageContainer/PageContainer.scss [hunk 1]
```diff
@@ -1,3 +1,3 @@
 .content-container {
   min-height: calc(100vh - 300px);
-}
```
Comment (jlajoie) [inline src/components/PageContainer/PageContainer.scss]: nit: don't need this change here

File: src/components/SidePanes/UsersPane.jsx [hunk 2]
```diff
@@ -119,6 +121,7 @@ class UsersPane extends React.Component {
             ))}
           </div>
         }
+        {!showEditors && organizationUsers.error && <div className="error-msg">{organizationUsers.error[0]}</div>}
```
Comment (jlajoie) [inline src/components/SidePanes/UsersPane.jsx]: We have a ```StatusAlert``` component I think we've used elsewhere for showing errors, might be worth just using that guy instead.

File: src/components/SidePanes/UsersPane.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.error-msg {
+  color: #FF0000;
```
Comment (jlajoie) [inline src/components/SidePanes/UsersPane.scss]: nit: I think typically we've just used bootstrap's color class names, but this whole file might go away if you swap to StatusAlert

File: src/components/SidePanes/UsersPane.jsx (hunk 2 above)
Comment (jlajoie) [reply src/components/SidePanes/UsersPane.jsx]: Also might be worth checking if the len of organizationUsers.error > 0 as well just for safety

File: src/components/SidePanes/UsersPane.scss (hunk 3 above)
Comment (xnick421) [reply src/components/SidePanes/UsersPane.scss]: ty on headsup with status alert component

**Pull request merged**